vector/src/main/java/fr/gouv/tchap/core/ui/views/HexagonMaskView.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package fr.gouv.tchap.core.ui.views
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Path
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatImageView
+import kotlin.math.sqrt
+
+class HexagonMaskView : AppCompatImageView {
```

Comment:
I know this class comes from the legacy app, but now that we are using Glide, a proper solution should be to extends BitmapTransformation and provide it to glide at rendering time.

Reply:
This will be handled later, see https://github.com/dinsic-pim/tchap-android-v2/issues/76